appengine/flexible/endpoints/README.md

```diff
@@ -93,3 +93,71 @@ To use the client ID for authentication:
 Now you can use the client ID to make requests to the API:
 
     $ python clients/google-id-token-client.py https://YOUR-PROJECT-ID.appspot.com YOUR-API-KEY /path/to/client-id.json
+
+### Using the App Engine default service account client (no key file needed)
+
+The App Engine default service account client demonstrates how to use the Google App Engine default service account to authenticate to endpoints.
+To use the client, you need to create a Google Cloud project as your client project.
```

Comment:
This sentence "To use the client, you need to create a Google Cloud project as your client project." reads a little vaguely. Can you try re-wording it?